src/mapcraftercore/renderer/renderviews/isometric/tilerenderer.cpp

```diff
@@ -194,17 +194,16 @@ void IsometricTileRenderer::renderTile(const TilePos& tile_pos, RGBAImage& tile)
 			// get local block position
 			mc::LocalBlockPos local(block.current);
 
-			// now get block id
-			uint16_t id = current_chunk->getBlockID(local);
+			// now get block id and block data
+			uint16_t id, data, extra_data;
+			current_chunk->getBlockInfo(local, id, data, extra_data);
```

Comment:
Performance-wise I thought it would be good to fetch the block ID first, and then the data if it's not air, but maybe it's not that relevant. Have you noticed any performance changes?